archive/tar/strconv.go

```diff
@@ -216,7 +216,7 @@ func parsePAXTime(s string) (time.Time, error) {
 	}
 
 	// Parse the nanoseconds.
-	if strings.Trim(sn, "0123456789") != "" {
+	if strings.TrimFunc(sn, func(r rune) bool { return r >= 0x30 && r <= 0x39 }) != "" {
```

Comment:
This could be `IndexFunc(…) != -1`, eliminating one more allocation — or, well, just a loop.

Reply:
Well, a secondary goal was to keep the patch minimal to help keep the next rebase simple, but you're right.  I'm not wedded to the specific change so much as the result, so if you want that change, I'm okay with it.

Reply:
Equally, there would be some benefit to minimizing the difference from original archive/tar …

It’s not at all up to me, I know nothing about the history of tar-split and almost nothing about its goals and philosophy.